### PR TITLE
[FEATURE] Excidium

### PR DESCRIPTION
**Pull request by Gyrandola**
## EXCIDIUM

From latin _massacre_, _slaughter_, the Excidium is a machine designed to allow anyone to put a bounty on someone else's head.

picture

**CREATING A BOUNTY**

Anyone can put a bounty on someone's head. To do so, one must interact with the Excidium and go through all the questions it will ask: name, reward, reason. With this information the Excidium announces the new bounty over SCOMM and updates the publicly available list of active bounties. Make sure to have enough funds in your SHYLOCK.

**COMPLETING A BOUNTY**

You decide to pursue a bounty. Very well, you only know the target's name and the reason why they are wanted dead, unless the creator was kind enough to give more information. Your job is to find the target and elegantly separate his head from the rest of his body, which you will present to the Excidium's maw for analysis. If the head is matches a bounty, the reward will be dispensed.

**EXCIDIUM LOCATION**
The Excidium is located in the central town plaza, in front of the guillotine.

**QUESTIONS**

Q: Why would the King allow this machine to exist? Murder is not nice.
A: The King benefits from sanctioned bounty hunting as a royal tithe of 10% is applied to each bounty reward.

Q: Can the machine be disabled?
A: Not as of now, but I intend to give the king or the steward this option.

image
                          _Example of bounties. The phrasing is generated randomly and filled with the real data._

image
_Example of the Excidium announcing the creation of a new bounty. It is also broadcasted to SCOM._
## Why It's Good For The Game

An indirect continuation of my mercenary rework PR, the Excidium incentivizes small-time contracts and mercenary work that anyone, alone or in a party, can pursue.

### CODE REVIEW

**Comment by BlueMemesauce** [comment]
bounties should be a global list
bandits should automatically get global bounties

File: code/modules/roguetown/roguemachine/bounty.dm [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+	var/amount
+	var/reason
+	var/employer
+
+	/// Whats displayed when consulting the bounties
+	var/banner
+
+/obj/structure/roguemachine/bounty/attack_hand(mob/user)
+
+	if(!ishuman(user)) return
```
Comment (BlueMemesauce) [inline code/modules/roguetown/roguemachine/bounty.dm:25]:
```suggestion
	if(!ishuman(user))
	 	return
```

**Comment by Real-MAGNUM** [comment]
Download the map from github and re-add the bounty machine please.

**Comment by Gyrandola** [comment]
> Download the map from github and re-add the bounty machine please.

hasn't really resolved it

File: code/modules/roguetown/roguemachine/bounty.dm (hunk 1 above)
Comment (Gyrandola) [reply code/modules/roguetown/roguemachine/bounty.dm:25]: does that not work?

Comment (BlueMemesauce) [reply code/modules/roguetown/roguemachine/bounty.dm:25]: no idea but either way its slop why would you put it like that

Comment (Gyrandola) [reply code/modules/roguetown/roguemachine/bounty.dm:25]: > no idea but either way its slop why would you put it like that

Slop? It's elegant

**Comment by Real-MAGNUM** [comment]
at this point just remove the changes done to blackstone.dmm I'll map it in myself.

**Comment by ophaq** [comment]
You should add a minimum mammon limit to this so people don't go spamming bounties for 1 mammon one all townsfolk and give people a real to kill each other "because it's funny to kill people for 1 mammon".

**Comment by Real-MAGNUM** [comment]
> You should add a minimum mammon limit to this so people don't go spamming bounties for 1 mammon on all townsfolk and give people a reason to kill each other "because it's funny to kill people for 1 mammon".

Yeah, that's a really good point. Minimum of 20-40 mammon.

**Comment by Gyrandola** [comment]
>  You should add a minimum mammon limit to this so people don't go spamming bounties for 1 mammon on all townsfolk and give people a reason to kill each other "because it's funny to kill people for 1 mammon".

> Yeah, that's a really good point. Minimum of 20-40 mammon.

Theres a minimum of 20 already

**Pull request merged**